### DISCUSSION TITLE
Cannot create team

### DISCUSSION THREAD

**Discussion by icomad** [vercel/community#1739] [Help] [answered]
### Summary

Hello, I am trying to create a team with the pro trial plan but it fails telling me "Error:Invalid request: `orgId` should be string."

### Additional information

<img width="487" alt="SCR-20230309-s94" src="https://user-images.githubusercontent.com/11632321/224132519-75e249e5-4448-4824-a9b9-f04c326d7350.png">

**Comment by diguini** [comment]
Same Problem here

![image](https://user-images.githubusercontent.com/62571987/224136229-ce5206b4-4e16-47cf-8caa-42a561803221.png)

**Comment by leerob** [comment, marked as answer]
Hey! Apologies for the bug introduced here, this was fixed short after it was reported. Should be working now 😄
